staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/kcp_rest_options_getter.go

```diff
@@ -41,10 +41,20 @@ func (t apiBindingAwareCRDRESTOptionsGetter) GetRESTOptions(resource schema.Grou
 		return ret, nil
 	}
 
-	ret.StorageConfig.KcpExtraStorageMetadata.Cluster = genericapirequest.Cluster{Wildcard: true}
-	// Normal CRDs (not coming from an APIBinding) are stored in e.g. /registry/mygroup.io/widgets/customresources/...
+	ret.StorageConfig.KcpExtraStorageMetadata.Cluster.Wildcard = true
+
+	// Normal CRDs (not coming from an APIBinding) are stored in e.g. /registry/mygroup.io/widgets/<cluster name>/...
 	if _, bound := t.crd.Annotations["apis.kcp.io/bound-crd"]; !bound {
 		ret.ResourcePrefix += "/customresources"
+
+		clusterName := logicalcluster.From(t.crd)
+		if clusterName != "system:system-crds" {
+			// For all normal CRDs outside of the system:system-crds logical cluster, tell the watch cache the name
```

Comment:
thanks for this comment but i'm afraid that i don't understand why `system:system-crds` is special :(

Reply:
The crds live in system:system-crds but the crs can be in any logical cluster.

Reply:
ok, so we will ever have only single instance of any CRD that lives in system:system-crds.